index.js

```diff
@@ -3104,7 +3104,7 @@
   def('trim',
       {},
       [$.String, $.String],
-      R.trim);
+      function(s) { return s.trim(); });
```

Comment:
Personally I'm for writing ES5 and documenting that fact; a user has the option of a shim if desired for the use case